Framework/CurveFitting/test/Functions/ProductQuadraticExpTest.h

```diff
@@ -199,7 +199,7 @@ to check that the results are equal.
                   LinearIncrementingAssignment(0, 0.1));
     FunctionDomain1DVector domain(xValues);
 
-    Mantid::CurveFitting::Jacobian jacobian(nResults, 5);
+    Mantid::API::BasicJacobian jacobian(nResults, 5);
```

Comment:
Kind of feels like this abstract base class should be changed to be called `IJacobian` or something instead and then call this just `Jacobian`, but perhaps that touches too much code to change?